src/components/checkboxes/index.md.njk

```diff
@@ -65,19 +65,25 @@ You can add hints to checkbox items to provide additional information about the
 
 {{ example({group: "components", item: "checkboxes", example: "hint", html: true, nunjucks: true, open: false, size: "s"}) }}
 
-### Conditionally revealing content
+### Conditionally revealing a related question
 
-You can conditionally reveal content when the user selects a particular checkbox, so they only see content when it’s relevant to them.
+You can ask the user a related question when they select a particular checkbox, so they only see the question when it's relevant to them.
 
-For example, you could reveal a phone number input when the user selects the 'Contact me by phone' option.
+This might make two related questions easier to answer by grouping them on the same page. For example, you could reveal a phone number input when the user selects the 'Contact me by phone' option.
 
 {{ example({group: "components", item: "checkboxes", example: "conditional-reveal", html: true, nunjucks: true, open: false, size: "xl"}) }}
 
-Keep it simple - if you need to add a lot of content, consider showing it on the next page in the process instead.
+Keep it simple. If the related question is complicated or has more than one part, show it on the next page in the process instead.
+
+Conditionally reveal questions only - do not show or hide anything that is not a question.
```

Comment:
```suggestion
You should only conditionally reveal questions - do not show or hide anything that is not a question.
```
Current wording is beautifully succinct - just suggesting this in case it adds some 'voice' and personalises the content for users.